operator/validator/controller.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"crypto/rsa"
 	"encoding/hex"
+	commons2 "github.com/bloxapp/ssv/protocol/v2/qbft"
```

Comment:
Could you please rename it from `commons2` to something else? E. g. `qbftprotocol`